odpf/optimus/runtime_service.proto

```diff
@@ -21,6 +21,7 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
         description: "Optimus server";
     }
     schemes: HTTP;
+    host: "localhost:9100/api";
```

Comment:
Originally commented by @ravisuhag : @scortier @kushsharma What is the need of this?
(Had rebased it to fix pipeline)